results.tex

```diff
@@ -3,19 +3,23 @@ \section{Results}
 \subsection{Nuclear Material Inventory}
 
 \Cref{tab:sim_result1} lists \gls{EU} material inventory in 2050.
+The materials continue to accumulate after 2050, but we are
+interested in the \gls{UNF} France can receive in the early 
+times of its transition, which is 2050. 
```

Comment:
This sentence is a little awkward. Perhaps (if it's even true) you might mean: "The materials continue to accumulate after 2050, but the \gls{UNF} France receives before 2050 is most impactful for the feasibility of the transition."

Reply:
much elegance. Thank you.